parsl/executors/status_handling.py

```diff
@@ -243,7 +243,7 @@ def send_monitoring_info(self, status: Dict) -> None:
         # Send monitoring info for HTEX when monitoring enabled
         if self.monitoring_radio:
             msg = self.create_monitoring_info(status)
-            logger.debug("Sending message {} to hub from job status poller".format(msg))
+            logger.debug("Sending block monitoring message: %s", repr(msg))
```

Comment:
Might as well let the formatting take care of the representation, no?  Consider:
```diff
- logger.debug("Sending block monitoring message: %s", repr(msg))
+ logger.debug("Sending block monitoring message: %r", msg)
```
Can't always do this (given TOCTOU and if the argument is subsequently changed), but I think here should be safe, no?

Reply:
so updated in 7f5b6e7